block.go

```diff
@@ -259,7 +263,12 @@ func writeMetaFile(dir string, meta *BlockMeta) error {
 	if err := f.Close(); err != nil {
 		return err
 	}
-	return renameFile(tmp, path)
+
+	if err := fileutil.Rename(tmp, path); err != nil {
```

Comment:
this one should also be Replace as in some cases the destination file might already exist

like in : https://github.com/prometheus/tsdb/blob/e46ec89de67d3d8fc91720ae3cba6f9c7a3320b0/block.go#L334

Reply:
ok.